selfdrive/test/process_replay/test_processes.py

```diff
@@ -150,11 +150,15 @@ def format_diff(results, ref_commit):
                       help="Updates reference logs using current commit")
   parser.add_argument("--upload-only", action="store_true",
                       help="Skips testing processes and uploads logs from previous test run")
+  parser.add_argument("--use-cache", action="store_true",
+                      help="Caches log files")
```

Comment:
the point of the environment variable is that this arg doesn't need to be added to every script

Reply:
Do you need to set it within a certain folder? 
From openpilot folder:
FILEREADER_CACHE=1 self-drive/test/test_processes....... 
didn't cache

Reply:
Nevermind. 
I ran "FILEREADER_CACHE='1' scons -j8 && selfdrive/test/process_replay/test_processes.py..."
needed  "scons -j8 && FILEREADER_CACHE='1' selfdrive/test/process_replay/test_processes.py..."